components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -323,12 +341,12 @@ subroutine crm_physics_init(state, pbuf2d, species_class)
    use phys_control,          only: phys_getopts, use_gw_convect
    use phys_grid,             only: get_ncols_p
    use crm_history,           only: crm_history_init
-   use cam_history,    only: addfld
-   use constituents,   only: apcnst, bpcnst, cnst_name, cnst_longname, cnst_get_ind
+   use cam_history,           only: addfld, hist_fld_active
+   use constituents,          only: apcnst, bpcnst, cnst_name, cnst_longname, cnst_get_ind
+   use constituents,          only: pcnst, cnst_get_ind
```

Comment:
Doesn't this line overwrite line 345?

Reply:
yea, we still access all the same module variables, so this is ok.